go/interpreter/lfvm/memory_test.go

```diff
@@ -47,3 +50,58 @@ func TestExpansionCosts(t *testing.T) {
 		}
 	}
 }
+
+func TestMemory_SetWord(t *testing.T) {
+
+	tests := map[string]struct {
+		offset uint64
+		memory []byte
+	}{
+		"expand memory": {},
+		"memory expansion not needed": {
+			offset: 32,
+			memory: make([]byte, 128),
+		},
+	}
+
+	for name, test := range tests {
+		t.Run(name, func(t *testing.T) {
+
+			ctxt := getEmptyContext()
+			m := NewMemory()
+			m.store = test.memory
+			// generate non zero bytes value
```

Comment:
is there any difference with `[]byte{31: 0x01} ` it will not be zeroed as well.

Reply:
this is what I meant to do, yes.

Reply:
well, the idea here was to have all bytes be non zero. doing `[]byte{31: 0x01}` will create a byte slice with 31 `0x0` and then one `0x01`